instagram_clone/spec/features/index_spec.rb

```diff
@@ -0,0 +1,22 @@
+# require 'rails_helper'
+#
+# feature 'Instructing visitors' do
+#   context 'A user is not signed in' do
+#     scenario 'Should request that a user logs in' do
+#       visit '/'
+#       expect(page).to have_content 'Please log in to start posting your photos'
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)